### ISSUE TITLE
Missing apps in the list

### ISSUE THREAD

**Issue by Nyconing** [ITVlab/Launch-On-Boot#14]
Sideloaded apps is missing in the list. Maybe bugs? Please fix. Thanks a lot.

**Comment by Fleker** [comment]
Right now it only searches for activities with the LEANBACK_LAUNCHER intent.

**Comment by Nyconing** [comment]
I see.... It will be great if there has a feature to search through all `android.intent.action.MAIN`.

Im not sure why it was searched only leanback intent, but **maybe** we can set an option for users to choose whether intent to be searched, and warn users about disclaimer.

**Comment by justice91423** [comment]
Yes please.  Been pulling my hair out trying to find a way to get it to launch the browser on wake.

**Comment by taimaiduc** [comment]
Hi 👋 Talent,
Can you update allow display all applications?
I really want to start my camera streaming application 
when my android box is restart. 

Thank you so much. 

Or can you show me which part of code I can make a PR 🙏🙏🙏

**Comment by Fleker** [comment]
i haven't worked on this project in years. There's probably a dozen different tooling updates that'll need to be made. But the method to get all applications is [getLauncherApps](https://github.com/ITVlab/Launch-On-Boot/blob/master/app/src/main/java/news/androidtv/launchonboot/MainActivity.java#L149).

**Comment by MuhannadYT** [comment]
Please fix this
